src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -59,7 +112,7 @@ const GeneralInfoStep = ({ btnsBox }) => {
             <AppTextField
               fullWidth
               label={t('common.labels.firstName')}
-              onBlur={handleBlur('firstName')}
+              onBlur={() => {}}
```

Comment:
Why did you remove the handleBlur function here?

Reply:
My bad, it’s file incorrect to this ticket and push, it doesn’t matter